lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -40,6 +40,14 @@ function subscribeEvents() {
     const label = args !== null ? '#' + args.attr('id') : '';
     ga('send', 'event', 'Modal', 'Close', label);
   });
+
+  // Signup button on pitch page
+  const urlParts = window.location.pathname.split('/');
+  if (urlParts[urlParts.length - 1] === 'pitch') {
+    $('#dosomething-signup-form-primary').on('submit', () => {
```

Comment:
Is this the ID for both the logged-in "sign up" button form and the logged-out "register and sign up" form? If not, will this still give us the results we're expecting for this test?